js/main.js

```diff
@@ -64,10 +64,11 @@ function renderEntry(entry) {
 }
 
 function handleDomContent(event) {
+  $noEntriesText.className = 'hidden';
+  swapViews(event);
```

Comment:
Your `swapViews` function has no parameter, so you don't need to pass it an argument